lib/sem.rb

```diff
@@ -39,6 +43,12 @@ def start(args)
 
     private
 
+    def on_resource_not_found(exception)
+      puts "[ERROR] Resource not found."
+      puts ""
+      puts "Resource not found: #{exception.message}"
```

Comment:
This is a bit too generic. Can we have more specific, nicer messages like, `[ERROR] User ijovan not found` ?